nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py

```diff
@@ -260,23 +260,27 @@ def setup_training_data(self, train_data_config: Optional[DictConfig] = None):
         if train_data_config is None:
             train_data_config = self._cfg.train_ds
 
+        if not hasattr(self._cfg, "class_labels") or self._cfg.class_labels is None:
```

Comment:
I would suggest to add a comment here that it is a fix for older (pre-1.0.0.b3) configs.

Reply:
done